lib/riemann/tools.rb

```diff
@@ -65,6 +65,7 @@ def report(event)
       end
       
       event = event.merge(attributes)
+      event[:time] = Integer(event[:time]) if event.has_key? :time
```

Comment:
Absolutely. What do you think about catching the exception and outputting some explanatory text, for others who may run into this later?